src/layout.html

```diff
@@ -30,6 +30,9 @@
         </svg>
       </button>
     </div>
+    <div class="footer" id="loader">
```

Comment:
@akshaymayekar do we need two footer divs? Can we move these two different elements (one is input and the other is a div lds-dual-ring) into one footer? I prefer to use classes instead of id's if you are adding them to use as selectors in javascript
document.querySelector(".footer .loader") should work